src/lib/helpers.ts

```diff
@@ -4,6 +4,7 @@ import { PETERPORTAL_GRAPHQL_ENDPOINT, PETERPORTAL_WEBSOC_ENDPOINT, WEBSOC_ENDPO
 import { Meeting, Section, WebsocResponse } from './peterportal.types';
 import { addCourse, openSnackbar } from '$actions/AppStoreActions';
 import AppStore from '$stores/AppStore';
+import {useMediaQuery} from "@material-ui/core";
```

Comment:
Where is this being used? This is the only line of diff in this file.

Reply:
Forgot to delete it, was trying something that didn't work.